package.json

```diff
@@ -34,7 +34,7 @@
     "jest": "^29.7.0",
     "postcss": "^8.4.27",
     "prettier": "3.0.0",
-    "tailwindcss": "^3.3.3",
+    "tailwindcss": "^3.4.3",
```

Comment:
Keep any package updates as a separate commit. **Example**: _Update tailwind_

Reply:
Duly noted, thank you!